.ci/win-ci-tools.psm1

```diff
@@ -125,10 +141,14 @@ Function DownloadFile {
         Try {
             $WebClient.DownloadFile($RemoteUrl, $DestinationPath)
             $Completed = $true
-        } Catch {
+        } Catch (WebException wex) {
+            If (((HttpWebResponse) wex.Response).StatusCode == HttpStatusCode.NotFound)
+            {
+            // error 404, do what you need to do
```

Comment:
@AlexNDRmac We need your experience with .Net here. Could you please take a look